### DISCUSSION TITLE
[Feature request] 有没有一种可能，md 文件可以支持中文命名？

### DISCUSSION THREAD

**Discussion by MoonLord-LM** [vuepress/core#1459] [Q&A] [answered]
### Clear and concise description of the problem

md 文件支持中文命名，方便在中文环境下查看和编辑

### Suggested solution

按 UTF-8 的 URL 编码，对文件名进行编码解码之类的处理就可以吧，大概

**Comment by meteorlxy** [comment, marked as answer]
为什么觉得不支持呢？

另：

- 在提 issue 的时候建议不要使用 ”有没有一种可能“
- 通常问号结尾的句子我们称为疑问句，请移步 Discussions 进行讨论

<img width="320" alt="image" src="https://github.com/vuepress/core/assets/18205362/bef61ee4-a5a5-4c54-9e0a-192032054176">
